docs/modules/ROOT/pages/how-tos/synthesize.adoc

```diff
@@ -0,0 +1,256 @@
++
+.<cloud>/cloudname.yml
+[source,yaml]
+----
+classes:
+  - global.cloud.<cloudname>.params
+  - global.cloud.<cloudname>.${facts:region}
+----
+
+* For a missing region, or if you're creating a new multi-region cloud, create file `cloud/<cloudname>/<regionname>.yml` in the repo.
+* For a missing distribution, create file `distribution/<distributionname>.yml` in the repo.
+
+.. "Rancher Cluster ID (Fact: rancher_cluster_id)": If it's a Rancher cluster, add the Cluster ID from the Rancher Management Server to this field
+.. "Repository Type": Leave at "auto" to have Lieutenant manage the cluster catalog repo on https://git.vshn.net
+.. "GitRepo URL": Leave empty if "Repository Type" is auto.
+   Check with the {projectsyn-ig}, if you have special requirements for the cluster catalog repo.
+
++
+[NOTE]
+====
+Not all combinations work smoothly.
+If in doubt, seek help from the {projectsyn-ig}.
+====
+
+. Add initial configuration in the `<cluster-id>.yml` file in the tenant's Project Syn configuration repository (often called the "Syn tenant repo").
+See {portal-lieutenant-tenants} for a list of tenants and their associated configuration repositories.
+
+. Run Commodore to generate the initial cluster catalog.
++
+Grab the Kubeconfig for the {synfra-cluster}
+Then you can setup the environment for Commodore as follows:
++
+[source,bash]
+----
+# Adjust this to point to where you've stored the Kubeconfig file for the Synfra cluster
+export KUBECONFIG=/path/to/synfra.kubeconfig
+export COMMODORE_API_TOKEN=$(kubectl config view -o jsonpath='{.users[?(@.name == "syn-synfra")].user.token}'  --raw)
+export COMMODORE_API_URL="https://api.syn.vshn.net"
+----
++
+Now you can follow the steps in the https://syn.tools/syn/how-tos/compile-catalog.html#_compilation[Project Syn "Compile a catalog" how-to]
+
+. Add required tokens to Vault.
+Check the documentation for the K8s distribution of your cluster for required secrets.
++
+All distributions require tokens for https://k8up.io[K8up] and the https://github.com/projectsyn/component-cluster-backup[cluster-backup] component. Make sure you've got the https://www.vaultproject.io/docs/install[Vault CLI] available locally and run the following commands:
++
+[source,bash]
+----
+export VAULT_ADDR=https://vault-prod.syn.vshn.net
+export TENANT_ID=<tenant id> <1>
+export CLUSTER_ID=<cluster id> <2>
+
+vault login -method=ldap username=<VSHN username> <3>
+
+# Configure global restic password for K8up
+vault kv put -cas=0 clusters/kv/${TENANT_ID}/${CLUSTER_ID}/global-backup password=$(pwgen -s 32 1)
+
+# Configure restic password for cluster backups
+vault kv put -cas=0 clusters/kv/${TENANT_ID}/${CLUSTER_ID}/cluster-backup password=$(pwgen -s 32 1)
+----
+<1> You can find the tenant ID on {portal-lieutenant-tenants}
+<2> The cluster ID which you chose in step 1a
+<3> Your usual VSHN LDAP login, generally `firstname.lastname`
++
+Check tenant-specific documentation for further tokens to configure in Vault.
++
+[NOTE]
```

Comment:
I don't think that this note belongs into this documentation.

Reply:
removed